docs/validation.rst

```diff
@@ -107,3 +109,35 @@ the form *variable* ``op`` *value* to the data in a NetCDF file, where
 predicate and a pass or fail message, and exits with a non-zero value
 if any of the predicates failed.
 
+Implementation notes
+--------------------
+
+The existing run scripts use a helper script ``scripts/model_common.sh``
+to assist in marshalling parameters and invoking particular model
+implementations. For a simulator ``SIM``, they in turn look for an
+implementation-specific script called ``run-SIM``, which expects
+command line arguments of the form:
+
+``run-SIM -o OUTPUT [--tag TAG]... [KEY=VALUE ...]``
+
+The simulator-specific run script is responsible for returning the
+unsupported tag error code if it does not support a requested tag.
+
+A python helper module ``nsuite.stdarg`` is intended to make parsing
+these options more straightforward. Similarly, the C++ header
+``validation/src/include/common_args.h`` is used for the Arbor
+model implementations.
+
+As much as is feasible, it is recommended that model implementations
```

Comment:
This could be simplified to "Where possible".

Reply:
I'd prefer to keep the original wording.